src/ripple/basics/base_uint.h

```diff
@@ -37,6 +37,20 @@
 
 namespace ripple {
 
+namespace detail
+{
+
+template <class Container>
+using contiguous_container
```

Comment:
How about naming this `is_contiguous_container` or `void_if_contiguous_container`?

Reply:
The `is_traits` typically evaluate to true or false.  And the `void_if` variant just seems wordy to me.